v19.2/build-a-python-app-with-cockroachdb-django.md

```diff
@@ -35,16 +35,26 @@ The example code and instructions on this page use Python 3 and Django 3.0.
 
 ## Step 1. Install Django and the CockroachDB backend for Django
 
-Install [Django](https://docs.djangoproject.com/en/3.0/topics/install/) and the [CockroachDB backend for Django](https://github.com/cockroachdb/django-cockroachdb):
+Install [Django](https://docs.djangoproject.com/en/3.0/topics/install/):
 
 {% include copy-clipboard.html %}
 ~~~ shell
-$ python3 -m pip install django==3.0.*
+$ python -m pip install django==3.0.*
 ~~~
 
+Before installing the [CockroachDB backend for Django](https://github.com/cockroachdb/django-cockroachdb), you must install one of the following psycopg2 prerequisites:
+
+- [psycopg2](https://pypi.org/project/psycopg2/), which has some
```

Comment:
nit: this bullet is wrapping (in markdown-- doesn't seem to be affecting how it renders though)